modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -556,14 +591,71 @@ private void _writeOutputFile(String fileName, String content, boolean rtl)
 
 	private static RTLCSSConverter _rtlCSSConverter;
 
-	private final boolean _appendCssImportTimestamps;
-	private final boolean _cleanPortalCommonDir;
-	private final String _docrootDirName;
-	private final boolean _generateSourceMap;
-	private final String _outputDirName;
-	private final String _portalCommonDirName;
-	private final int _precision;
-	private final Pattern[] _rtlExcludedPathPatterns;
+	@Parameter(
```

Comment:
Why are you adding all this here, while we already have `CSSBuilderArgs`? Shouldn't you add the JCommander annotations there?